src/Swarm/TUI/View.hs

```diff
@@ -334,6 +308,37 @@ drawGameUI s =
   fr = s ^. uiState . uiFocusRing
   moreTop = s ^. uiState . uiMoreInfoTop
   moreBot = s ^. uiState . uiMoreInfoBot
+  showREPL = s ^. uiState . uiShowREPL
+  rightPanel = if showREPL then worldPanel ++ replPanel else worldPanel
+  worldPanel =
+    [ panel
+        highlightAttr
+        fr
+        (FocusablePanel WorldPanel)
+        ( plainBorder
+            & bottomLabels . rightLabel ?~ padLeftRight 1 (drawTPS s)
+            & topLabels . leftLabel ?~ drawModalMenu s
+            & addCursorPos
+            & addClock
+        )
+        (drawWorld (s ^. uiState . uiShowRobots) (s ^. gameState))
+    , drawKeyMenu s
```

Comment:
We should also add a hint about the keybinding to hide the REPL to the top line of the key menu.